packages/ibm-products/src/components/Datagrid/types/index.ts

```diff
@@ -266,6 +278,10 @@ export interface DataGridState<T extends object = any>
   loadMoreThreshold?: number;
   onRowClick?: (row, event) => void;
   onSort?: boolean;
+  customizeColumnsProps?: {
```

Comment:
do we need to add the below 2 properties to `customizeColumnProps` definition as we are setting those as well in the hook?
`isTearsheetOpen`,
`setIsTearsheetOpen`,

Reply:
Updated the props.